File: birdmeal/src/main/java/com/backend/birdmeal/dto/DonationHistoryDto.java
package com.backend.birdmeal.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class DonationHistoryDto {
    private Long donationHistorySeq;
    private Long userSeq;
    private int donationPrice;
    private String donationDate;
    private boolean donationType;
}

File: AOS/BirdMeal/app/src/main/java/com/ssafy/birdmeal/model/dto/DonationHistoryDto.kt
package com.ssafy.birdmeal.model.dto

import com.google.gson.annotations.SerializedName

data class DonationHistoryDto(
    @SerializedName("donationSeq") val donationSeq: Int = 0,
    @SerializedName("userSeq") val userSeq: Int = 0,
    @SerializedName("userNickname") val userNickname: String = "",
    @SerializedName("donationDate") val donationDate: String = "",
    @SerializedName("donationPrice") val donationPrice: Long = 0,
    @SerializedName("donationType") val donationType: Boolean = false,
)